moj-node/server/routes/timetable.js

```diff
@@ -0,0 +1,47 @@
+const express = require('express');
+const { path } = require('ramda');
+const { format, addDays } = require('date-fns');
+
+const today = new Date();
+
+module.exports = function Home({ logger, offenderService }) {
+  const router = express.Router();
+
+  router.get('/', async (req, res, next) => {
+    try {
+      logger.info('GET timetable');
+
+      const { notification } = req.session;
+      const userDetails = path(['session', 'user'], req);
+      const bookingId = path(['bookingId'], userDetails);
+      const todayDateString = format(today, 'yyyy-MM-dd');
+      const sevenDaysDateString = format(addDays(today, 6), 'yyyy-MM-dd');
+
+      const startDate = todayDateString;
+      const endDate = sevenDaysDateString;
```

Comment:
This will change as it will need to be a separate function when we know how we are navigating the timetable.